src/lib/features/export-import-toggles/export-import-controller.ts

```diff
@@ -74,7 +73,9 @@ class ExportImportController extends Controller {
                         200: createResponseSchema('exportResultSchema'),
                         ...getStandardResponses(404),
```

Comment:
If you don't provide a key you may probably get 401 - I'd test it in the openapi UI